packages/@aws-cdk/aws-glue-alpha/README.md

```diff
@@ -17,580 +17,242 @@
-  securityGroups: [securityGroup],
-  // The VPC subnet which contains the data source
-  subnet,
-});
-```
-
-For RDS `Connection` by JDBC, it is recommended to manage credentials using AWS Secrets Manager. To use Secret, specify `SECRET_ID` in `properties` like the following code. Note that in this case, the subnet must have a route to the AWS Secrets Manager VPC endpoint or to the AWS Secrets Manager endpoint through a NAT gateway.
-
-```ts
-declare const securityGroup: ec2.SecurityGroup;
-declare const subnet: ec2.Subnet;
-declare const db: rds.DatabaseCluster;
-new glue.Connection(this, "RdsConnection", {
-  type: glue.ConnectionType.JDBC,
-  securityGroups: [securityGroup],
-  subnet,
-  properties: {
-    JDBC_CONNECTION_URL: `jdbc:mysql://${db.clusterEndpoint.socketAddress}/databasename`,
-    JDBC_ENFORCE_SSL: "false",
-    SECRET_ID: db.secret!.secretName,
-  },
-});
-```
-
-If you need to use a connection type that doesn't exist as a static member on `ConnectionType`, you can instantiate a `ConnectionType` object, e.g: `new glue.ConnectionType('NEW_TYPE')`.
-
-See [Adding a Connection to Your Data Store](https://docs.aws.amazon.com/glue/latest/dg/populate-add-connection.html) and [Connection Structure](https://docs.aws.amazon.com/glue/latest/dg/aws-glue-api-catalog-connections.html#aws-glue-api-catalog-connections-Connection) documentation for more information on the supported data stores and their configurations.
-
-## SecurityConfiguration
-
-A `SecurityConfiguration` is a set of security properties that can be used by AWS Glue to encrypt data at rest.
-
-```ts
-new glue.SecurityConfiguration(this, 'MySecurityConfiguration', {
-  cloudWatchEncryption: {
-    mode: glue.CloudWatchEncryptionMode.KMS,
-  },
-  jobBookmarksEncryption: {
-    mode: glue.JobBookmarksEncryptionMode.CLIENT_SIDE_KMS,
-  },
-  s3Encryption: {
-    mode: glue.S3EncryptionMode.KMS,
-  },
-});
-```
-
-By default, a shared KMS key is created for use with the encryption configurations that require one. You can also supply your own key for each encryption config, for example, for CloudWatch encryption:
-
-```ts
-declare const key: kms.Key;
-new glue.SecurityConfiguration(this, 'MySecurityConfiguration', {
-  cloudWatchEncryption: {
-    mode: glue.CloudWatchEncryptionMode.KMS,
-    kmsKey: key,
-  },
-});
-```
-
-See [documentation](https://docs.aws.amazon.com/glue/latest/dg/encryption-security-configuration.html) for more info for Glue encrypting data written by Crawlers, Jobs, and Development Endpoints.
-
-## Database
-
-A `Database` is a logical grouping of `Tables` in the Glue Catalog.
-
-```ts
-new glue.Database(this, 'MyDatabase', {
-  databaseName: 'my_database',
-  description: 'my_database_description',
-});
-```
-
-## Table
-
-A Glue table describes a table of data in S3: its structure (column names and types), location of data (S3 objects with a common prefix in a S3 bucket), and format for the files (Json, Avro, Parquet, etc.):
```

Comment:
How come the documentation for `Database`, `Table` and many others are being deleted here? The code files are not being modified, is it still WIP or will they be removed or something?

Reply:
Discussed with @natalie-white-aws and the removal of these docs are unintended and will be updated to add them back.